develop/learning-paths/migration/articles/xx-theme/04-updating-the-resources-importer.markdown

```diff
@@ -0,0 +1,365 @@
+	    live out your childhood dream of being an astronaut on the Moon. If 
+	    that's not enough incentive, you'll enjoy a luxurious 3 day 2 night 
+	    stay in our fabulous Lunar accomodations. Enjoy a round of Lunar 
+	    Golf on our one of a kind course. Have a blast on our Rover Racing 
+	    track. Make your reservation now. The rest of your life starts 
+	    today!</div>
+    </div>
+    ]]>
+		    </dynamic-content>
+	    </dynamic-element>
+    </root>
+
+You may have also noticed that the `<div>` classes have been changed. The 
+`2 column description.xml` article shown above illustrates the updated Bootstrap 
+grid system for  Bootstrap 3. Instead of using the `span-[number]` class to 
+designate a column width, the `col-md-[number]` class is now used to designate 
+column width. The total amount for the column widths must still add up to 12.
+
+That's all that is needed for most of the basic web content articles. To make 
+the article migration run as smoothly as possible, you can replace the HTML web 
+content articles with the updated XML ones in the 
+`resources-importer/journal/articles/Basic Web Content/` directory of the files 
+you downloaded at the beginning.
+
+There is one last update you'll need to make to one of the web content articles
+which you can read about more next.
+
+### Updating Bootstrap to Lexicon
+
+Liferay 7's UI is built with a design language, known as [Lexicon](http://liferay.github.io/lexicon/).
+When building UI, it is recommended that you use Lexicon to follow the design
+patterns that Liferay has created.
+
+The reservation form for the lunar resort theme used Bootstrap's design language
+for the 6.2 themes Learning path. Although Lexicon is an extension of Bootstrap, 
+it uses its own design patterns and classes that you shoudl be aware of. To 
+follow best practices for Liferay 7, this form should be updated to follow 
+Lexicon's design language.
+
+Follow the step below to update the reservation form:
+
+1.  Open the `Reservation Form.xml` article in the 
+    `/src/resources-importer/journal/articles/Basic Web Content/` directory and
+    replace the contents with the markup below:
+    
+    <?xml version="1.0"?>
+    
+    <root available-locales="en_US" default-locale="en_US">
+            <dynamic-element name="content" type="text_area" 
+            index-type="keyword" index="0">
+                    <dynamic-content language-id="en_US">
+                            <![CDATA[
+                                    <p>Thanks for choosing to stay at the 
+                                    Liferay Lunar Resort! Please fill out the 
+                                    form below to book your stay. We know you 
+                                    have a choice in where to stay on the Moon... 
+                                    oh wait no you don't. Thanks for picking us 
+                                    anyways. We'll see you soon on the Moon!</p>
+    <form role="form-horizontal">
```

Comment:
This section name might be a bit of a misnomer, since the new markup for the Reservation Form is actually just updated Bootstrap markup, none of it is Lexicon specific. http://getbootstrap.com/css/#forms